### PR TITLE
Wiktionary changes only

### PR DESCRIPTION
**Pull request by scarlettmparker**
- Added english support for the wiktionary command (works for both english and greek words),
- Fixed the remove_ancient_greek function to prevent the bot from returning nothing where ancient greek and greek entry is reversed in the english wiktionary.
- Wiktionary messages now get split up so the character limit isn't hit and some fields have been removed from the greek list so you don't get overwhelmed with information

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 from discord import app_commands
 from dotenv import dotenv_values
+
+import pronunciation.pronunciation as pronunciation
 from gr_datetime.gr_date import get_full_date
 from help.help import HelpMessage
 from utils import Pagination, fix_greek_spelling
-from wordlookup.wiktionaryel import fetch_conjugation
+from wordlookup.embed_message import embed_message as wiktionary_message
+from wordlookup.wiktionaryel import fetch_conjugation, fetch_wiktionary_pos
```
Comment (daxida) [inline src/main.py]: `fetch_wiktionary_pos` is not used

File: src/main.py [hunk 2]
```diff
@@ -21,10 +23,6 @@ async def on_ready(self) -> None:
             self.synced = True
         print(f"\033[32mBot is ready! {self.user}\033[0m")
 
-    async def on_message(self, message: discord.Message) -> None:
```
Comment (daxida) [inline src/main.py]: What is the reason for removing this?

File: src/main.py [hunk 3]
```diff
@@ -63,6 +61,42 @@ async def template_command(
...
+):
+    ephemeral = ephemeral.lower() in ["true", "yes", "1"]
+    embeds = await wiktionary_message(word, language)
+
+    if len(embeds) == 1:
+        await interaction.response.send_message(embed=embeds[0], ephemeral=ephemeral)
+    else:
+        await interaction.response.send_message(embed=embeds[0], ephemeral=ephemeral)
+        for embed in embeds[1:]:
+            await interaction.followup.send(embed=embed, ephemeral=ephemeral)
```
Comment (daxida) [inline src/main.py]:
```suggestion
     await interaction.response.send_message(embed=embeds[0], ephemeral=ephemeral)
     if len(embeds) > 1:
        for embed in embeds[1:]:
            await interaction.followup.send(embed=embed, ephemeral=ephemeral)
```

File: src/wordlookup/wiktionaryel.py [hunk 4]
```diff
@@ -56,28 +65,43 @@ async def create(cls, word: str):
...
-            for sibling in parent.find_next_siblings():
-                sibling.extract()
+            current_element = tag_to_remove.find_parent()
+            while current_element:
+                next_sibling = current_element.find_next_sibling()
+                if next_sibling and stop_at_string and next_sibling.find("h2", string=stop_at_string):
+                    break
+
+                # remove the current element and move to next sibling
+                current_element.extract()
```
Comment (daxida) [inline src/wordlookup/wiktionaryel.py:95]: Does this line do anything?

File: src/wordlookup/wiktionaryel.py [hunk 5]
```diff
@@ -43,11 +47,16 @@ class WiktionaryQuery:
     __slots__ = "word", "soup"
 
     @classmethod
-    async def create(cls, word: str):
+    async def create(cls, word: str, language: str):
```
Comment (daxida) [inline src/wordlookup/wiktionaryel.py:50]: You probably want language as an attribute of the instance instead of passing it to every method later on.

**Review by daxida — CHANGES_REQUESTED** [review]

File: src/main.py (hunk 2 above)
Comment (scarlettmparker) [reply src/main.py]: whoops i was removing some lines when extracting only the wiktionary changes from the faq and must have accidentally removed it. my bad

File: src/wordlookup/wiktionaryel.py (hunk 4 above)
Comment (scarlettmparker) [reply src/wordlookup/wiktionaryel.py:95]: yeah it removes the ancient greek entries so you don't get a muddle of ancient greek and modern greek entries. the break is to stop the modern greek entries being removed on the en pages as the order is switched.

**Pull request merged**